test/images/agnhost/webhook/crd.go

```diff
@@ -19,32 +19,31 @@ package webhook
 import (
 	"fmt"
 
+	"k8s.io/api/admission/v1"
 	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"k8s.io/api/admission/v1beta1"
 	"k8s.io/klog"
 )
 
-// This function expects all CRDs submitted to it to be apiextensions.k8s.io/v1beta1
+// This function expects all CRDs submitted to it to be apiextensions.k8s.io/v1
 // TODO: When apiextensions.k8s.io/v1 is added we will need to update this function.
-func admitCRD(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
+func admitCRD(ar v1.AdmissionReview) *v1.AdmissionResponse {
 	klog.V(2).Info("admitting crd")
-	crdResource := metav1.GroupVersionResource{Group: "apiextensions.k8s.io", Version: "v1beta1", Resource: "customresourcedefinitions"}
+	crdResource := metav1.GroupVersionResource{Group: "apiextensions.k8s.io", Version: "v1", Resource: "customresourcedefinitions"}
```

Comment:
this isn't correct, this is the version of the CRD being admitted. surprised this passed local testing

Reply:
I had only spot checked the api with curl. Clearly that was insufficient. I'll try to figure out how to run e2e against it.

Reply:
Published agnhost image to personal repo and kicked off e2e with `test/utils/image/manifest.go` updated to point to personal repo. Waiting for tests to run.

Reply:
Update: e2e passed with agnhost image from this PR. Also updated `AdmissionReviewVersions` to be `v1` and re-ran the e2e tests but got errors like `ValidatingWebhookConfiguration.admissionregistration.k8s.io \"webhook-5090blocking\" is invalid: webhooks[0].admissionReviewVersions: Invalid value: []string{\"v1\"}: must include at least one of v1beta1`, which I didn't expect with https://github.com/kubernetes/kubernetes/pull/80231 merged before this PR was branched.

Reply:
validation requires at least one version recognized by the previous API server to protect skewed HA clusters

Reply:
If you want to get v1 admissionreviews, do this: `admissionReviewVersions: ["v1","v1beta1"]`

that indicates the webhook can handle v1beta1 review types (which is required for one more release), but prefers v1 review types

Reply:
Thanks! e2e test found that we were missing GVK in AdmissionReview responses. I've added it and am rerunning e2e now.